cleo/commands/list_command.py

```diff
@@ -22,12 +23,23 @@ class ListCommand(Command):
     arguments = [
         Argument("namespace", required=False, description="The namespace name")
     ]
+    options = [
+        option("version", "V", "Display this application version.", flag=True),
+    ]
 
     def handle(self) -> int:
+        if self.option("version"):
+            self.io.write_line(self.application.long_version)
+
+            return 0
+
         from cleo.descriptors.text_descriptor import TextDescriptor
 
         TextDescriptor().describe(
-            self._io, self.application, namespace=self.argument("namespace")
+            self._io,
+            self.application,
+            namespace=self.argument("namespace"),
+            definition=self.definition,
```

Comment:
Since `application` only has the default definition, I ended up passing `definition` as an extra argument, to retrieve it in `_describe_application`.
Otherwise, the command would not display `--version` option.

If there's a better way to handle that, I would gladly update that.